apps/nova/src/components/Chat/Rooms.tsx

```diff
@@ -53,6 +55,10 @@ const Rooms: React.FC<{
           <ChatRoom
             key={room.roomId}
             isActive={room.roomId === roomId}
+            readOnly={!!room.latestMessage}
```

Comment:
I find this semantically hard to understand. We may rename it to `showOptions` instead of `readOnly`. Or `optionsEnabled` would be better as the former indicates that it's a function parameter (that will show options). Furthermore, why users are not allowed to see/use the options unless they read the latest message?

Reply:
nice naming suggestion, Chat rooms is a generic component that will be used in showing the pinned, all messages and the rooms with tutors that user hasn't contacted yet.

in last case, we wouldn't show the menu. (user can't do room actions as these aren't real rooms)